### ISSUE TITLE
eigenvalue density is the same for chol_polar even with the bug from issue #45

### ISSUE THREAD

**Issue by irenecrsn** [irenecrsn/gmat#48]
I didn't expect this.

**Comment by gherardovarando** [comment]
I do not think it is a bug. 
I never thought that **just** looking at the eigenvalue distribution was enough. 
I was just using it as a first check.
Bu anyway it is interesting to know, and probably we could explore it further.

**Comment by irenecrsn** [comment]
Yes you are right, it is not a bug. It is surprising thought that even though the matrices are entirely different, they have the same eigenvalues. Probably there is some result out there about which operations preserve them, and `t/anti_t` may be one of them.
